internal/observability/event/event.go

```diff
@@ -12,40 +12,39 @@ type (
 // RequestInfo defines the fields captured about a Boundary request.
 type RequestInfo struct {
 	EventId  string `json:"-"`
-	Id       string `json:"id,omitempty"`
-	Method   string `json:"method,omitempty"`
-	Path     string `json:"path,omitempty"`
-	PublicId string `json:"public_id,omitempty"`
+	Id       string `json:"id,omitempty" class:"public"`
+	Method   string `json:"method,omitempty" class:"public"`
+	Path     string `json:"path,omitempty" class:"public"`
+	PublicId string `json:"public_id,omitempty" class:"public"`
 }
 
 // UserInfo defines the fields captured about a user for a Boundary request.
 type UserInfo struct {
-	UserId        string `json:"id,omitempty"`
-	AuthAccountId string `json:"auth_account_id,omitempty"`
+	UserId        string `json:"id,omitempty" class:"public"`
+	AuthAccountId string `json:"auth_account_id,omitempty" class:"public"`
 }
 
 type GrantsInfo struct {
 	Grants []GrantsPair `json:"grants_pair,omitempty"`
 }
 
 type GrantsPair struct {
-	Grant   string `json:"grant,omitempty"`
-	ScopeId string `json:"scope_id,omitempty"`
+	Grant   string `json:"grant,omitempty" class:"public"`
+	ScopeId string `json:"scope_id,omitempty" class:"public"`
 }
 
 type Auth struct {
-	// AccessorId is a std audit field == auth_token public_id
-	AccessorId string      `json:"accessor_id"`
-	UserInfo   *UserInfo   `json:"user_info,omitempty"` // boundary field
-	GrantsInfo *GrantsInfo `json:"grants_info,omitempty"`
-	UserEmail  string      `json:"email,omitempty"`
-	UserName   string      `json:"name,omitempty"`
+	AuthTokenId string      `json:"auth_token_id" class:"public"`
+	UserInfo    *UserInfo   `json:"user_info,omitempty"` // boundary field
+	GrantsInfo  *GrantsInfo `json:"grants_info,omitempty"`
+	UserEmail   string      `json:"email,omitempty" class:"sensitive"`
+	UserName    string      `json:"name,omitempty" class:"sensitive"`
 }
 
 type Request struct {
-	Operation string        `json:"operation"` // std audit field
-	Endpoint  string        `json:"endpoint"`  // std audit field
-	Details   proto.Message `json:"details"`   // boundary field
+	Operation string        `json:"operation,omitempty" class:"public"` // std audit field
+	Endpoint  string        `json:"endpoint,omitempty" class:"public"`  // std audit field
```

Comment:
What's the difference between endpoint here and path in requestinfo?

Reply:
Great question... and I don't think there is any difference.  Aligning on what fields should be audited is still outstanding and your comment reminds me to restart that effort.  I'm going to dig up Sarah's RFC which referenced this and perhaps try to get alignment via that.  With all that said, I believe this `Request` struct is included in Kit's work that attempts to standarize and align upon a common set of data fields and format for HashiCorp products.